components/callbacks.py

```diff
@@ -226,16 +225,17 @@ def off_on_topic(update: Update, context: CallbackContext) -> None:
                 parse_mode=ParseMode.MARKDOWN,
```

Comment:
lets use `reply_markdown` and replace pythontelegrambottalk with `{OFFTOPIC_USERNAME}`